clickhouse/client.cpp

```diff
@@ -64,8 +64,12 @@ struct ClientInfo {
 };
 
 std::ostream& operator<<(std::ostream& os, const ClientOptions& opt) {
-    os << "Client(" << opt.user << '@' << opt.host << ":" << opt.port
-       << " ping_before_query:" << opt.ping_before_query
+    os << "Client(";
```

Comment:
That ignores `opt.host` and `opt.port` completely, and this `operator<<` might be used on unmodified `ClientOptions` (by tests or users), making output confusing.